pt.html

```diff
@@ -201,96 +207,97 @@
 </words>
 <words x=60 y=300 w=400 h=60>
-	...and this lil' memory card game became a core part of my <i>life</i>.
+	...e este pequeno jogo de cartas de memória tornou-se uma parte essencial da minha <i>vida</i>.
 </words>
 </panel>
 
 <panel w=550 h=350>
 <pic src="pics/intro0.png" sx=0 sy=850></pic>
 <words x=10 y=10 w=500 h=30>
-	In short, Spaced Repetition = testing + time.
+	Resumidamente, Repetição Espaçada = ensaio + tempo
 </words>
 <words x=10 y=220 w=500 h=90>
-	You test yourself on a fact repeatedly, spacing out your repetitions over time.
-	(But won't this take forever? Ah, as we'll see later, there's a trick...)
+	Você se testa em um fato repetidamente, espaçando suas repetições ao longo do tempo.
+	(Mas isso não vai demorar para sempre? Ah, como veremos mais tarde, há um truque...)
 </words>
 </panel>
 
 <panel w=400 h=400>
 <pic src="pics/intro0.png" sx=0 sy=1200></pic>
 <words x=10 y=20 w=290 no-bg>
-	Spaced Repetition is free, evidence-based, and so simple you can do it with a <i>shoebox</i>.
+	Repetição Espaçada é gratuita, baseada em evidências, e tão simples que você pode fazê-lo com uma <i>caixa de sapatos</i>.
 </words>
 <words x=308 y=265 w=50 no-bg class="comic_text" style="text-align:left; font-size:20px">
-	nicky shut up
+	nicky cala boca
 </words>
 <words x=90 y=130 w=270 no-bg>
-	So, what's the catch? Why isn't <i>everyone</i> already doing Spaced Repetition?
+	Então, qual é o truque? Por que nem <i>todo mundo</i> já está fazendo Repetição Espaçada?
 </words>
 </panel>
 
 <panel w=400 h=400>
 <pic src="pics/intro0.png" sx=400 sy=1200></pic>
 <words x=10 y=20 w=330 no-bg>
-	Well, the catch is that making <i>any</i> new habit is hard
-	– especially a weird habit like Spaced Repetition.
+	Bem, a pegadinha é que fazer <i>qualquer</i> novo hábito é difícil
+	- especialmente um hábito estranho como Repetição Espaçada.
 </words>
 <words x=292 y=303 w=80 no-bg class="comic_text" style="text-align:left; font-size:20px">
-	teacher's coming
+	professor está vindo
 </words>
 <words x=100 y=134 w=270 no-bg>
-	That's why I made this badly-drawn interactive comic.
+	É por isso que fiz esse quadrinho interativo mal desenhado.
 </words>
 </panel>
 
 <panel w=400 h=400>
 <pic src="pics/intro0.png" sx=800 sy=1200></pic>
 <words x=10 y=20 w=300 no-bg>
-	In this comic on Spaced Repetition,
-	I'll show you WHY it works, HOW it works...
+	Nestes quadrinhos em Repetição Espaçada,
+	vou mostrar POR QUE funciona, COMO funciona...
 </words>
 <words x=110 y=140 w=250 no-bg>
-	...and help you get started with it <i>TODAY.</i>
+	... e ajudar você a começar com isso <i>HOJE</i>.
 </words>
 </panel>
 
 <panel w=600 h=180>
 <words w=600 x=-15 no-bg>
 
-	Also, throughout this comic,
-	you can test yourself on what you've learnt,
-	at spaced-out intervals.
-	<b>That is: you'll use Spaced Repetition to learn about Spaced Repetition.</b>
+	Além disso, ao longo deste quadrinho,
+	você pode testar o que aprendeu
+	em intervalos regulares.
```

Comment:
Os intervalos não são `regulares`!

```suggestion
	em intervalos espaçados.
```